test_artifacts/cert.pem

```diff
@@ -0,0 +1,13 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
I think I'd rather have the contents of the `test_artifacts` folder be simply in `tests`, or maybe `tests/assets`